tasks/models.py

```diff
@@ -53,6 +53,9 @@ def filter_by_symbol(cls, priority_filter):
         print(priority_filter)
         return cls.objects.filter(priority=priority_filter)
 
+    def change_assignee(self, new_assignee):
+        self.assignee = new_assignee
```

Comment:
What will happen if the new_assignee does not exist?

Reply:
Good point. Made some changes. Uploading the PR right now.
Anyway, there are some cases that Django raises an exception, and we can use it.

Reply:
I wouldn't prevent task to move from team to team, but this is a business decision and I'll leave it to you.

But what if the new_assignee is not in the DB? Not part of the system?

Reply:
Is it possible? I mean when using the create_user or User.objects.create the user is saved in the DB.

Reply:
The implementation is simple here. But as I understand, once I create a user, it is saved in DB.
So I'm not sure how to write a test for this one

Reply:
@nunnatsa You can't assign a task to someone who is not exist, when the manager will edit the task he could only chose members of his team (which must be existing)

Reply:
Resolved with the last commit.